### ISSUE TITLE
Support for Knative Eventing

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#1505]
Knative Events are meant to provide "Universal subscription, delivery and management of events" for Knative/Kubernetes which could be connected into Akka Streams with Alpakka.

https://github.com/knative/eventing

**Comment by huntc** [comment]
Wouldn’t this be more about showing how Akka http, Alpakka AMQP, MQTT etc can be used in the context of knative events?

**Comment by ennru** [comment]
Yes, examples showing that would be great. I know too little about Knative yet, to understand what Alpakka can help you with there.
